src/components/EventDetails/EventHeader.js

```diff
@@ -145,14 +155,14 @@ const EventIcon = styled(Icon)`
 `;
 
 const EventImage = styled(CachedImage)`
-  width: 58px;
-  height: 58px;
+  width: ${props => props.imageDiameter || 58}px;
+  height: ${props => props.imageDiameter || 58}px;
 `;
 
 const ImageTouchable = styled.TouchableOpacity`
-  width: 58px;
-  height: 58px;
-  border-radius: 29px;
+  width: ${props => props.imageDiameter || 58}px;
+  height: ${props => props.imageDiameter || 58}px;
+  border-radius: ${props => props.imageDiameter ? props.imageDiameter / 2 : 29}px;
```

Comment:
```suggestion
  border-radius: ${props => props.imageDiameter / 2 || 29}px;
```

Reply:
imageDiameter could not be passed in as it is not required, thus reducing this line might cause crashes (NaN px as a border-radius value)

Reply:
It won't, NaN is false so it would be 29, but in this case there's another piece you want to consider changing this (it has my suggested approach):

https://github.com/pillarwallet/pillarwallet/pull/1506#pullrequestreview-311693976

to both lines match either one or another approach.

Reply:
Yeah, you are right. I take that back :) and will update this part. Thanks :)